app/src/test/java/com/example/sharingang/shake/ShakeDetectorTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.sharingang.shake
+
+import org.junit.Test
+
+class ShakeDetectorTest {
+    @Test
+    fun noShakeIsDetectedForSmallMovements() {
+        val detector = ShakeDetector()
+        assert(!detector.detect(0F, 0F, 0F, 0))
+        assert(!detector.detect(0F, 0F, 0F, 10000))
+    }
```

Comment:
Does this test case cover the case when there is no movement at all? I think you could maybe add a test case for when there is movement but not enough to trigger the shake condition.

Reply:
Yeah, not a bad idea for small movement